app/assets/javascripts/katello/hosts/activation_key_edit.js

```diff
@@ -26,17 +27,14 @@ function ktLoadActivationKeys() {
             $.each(response['results'], function (i, key) {
                 KT.hosts.availableActivationKeys[key.name] = [];
             });
-            ktAkUpdateSubscriptionsInfo();
+            tfm.typeAheadSelect.updateOptions(Object.keys(KT.hosts.availableActivationKeys), KT.KT_AK_LABEL);
+        },
+        error: function() {
+          $("#ak-load-error").show();
         },
-        error: ktErrorLoadingActivationKeys
     });
 }
 
-function ktErrorLoadingActivationKeys(error) {
-    $.jnotify("Error while loading activation keys from Katello", { type: "error", sticky: true });
```

Comment:
jnotify has been gone from Foreman for a while, so I changed how we show an error